components/camel-jmx/pom.xml

```diff
@@ -48,6 +48,16 @@
             <artifactId>camel-management</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>jakarta.xml.bind</groupId>
+            <artifactId>jakarta.xml.bind-api</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.sun.xml.bind</groupId>
```

Comment:
Is there not a jakarta jaxb impl class we can use?

Reply:
I think this one is part of the Jakarta implementation after 2.3.1 (https://eclipse-ee4j.github.io/jaxb-ri/). I am on mobile now, but I will double check later what version we are using. Maybe we have an opportunity to upgrade.

Reply:
The link should have been: https://eclipse-ee4j.github.io/jaxb-ri/2.3.3/docs/ch02.html